packages/m/mariadb-connector-c/xmake.lua

```diff
@@ -0,0 +1,107 @@
+package("mariadb-connector-c")
+    set_homepage("https://github.com/mariadb-corporation/mariadb-connector-c")
+    set_description("MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.")
+    set_license("LGPL-2.1")
+    add_urls("https://github.com/mariadb-corporation/mariadb-connector-c/archive/refs/tags/v$(version).tar.gz")
+    add_versions("3.1.13", "361136e9c365259397190109d50f8b6a65c628177792273b4acdb6978942b5e7")
+    add_deps("cmake")
+
+    add_links("mariadb")
+    add_linkdirs("lib/mariadb/")
+
+    -- Windows specific options
+    if is_plat("windows") then
+	    add_configs("iconv", {
+            description = "Enables character set conversion.", 
+            default = false, 
+            type = "boolean"
+	    })
+	    add_configs("msi", {
+            description = "Build MSI installation package.", 
+            default = false, 
+            type = "boolean"
+	    })
+	    add_configs("rtc", {
+            description = "Enables runtime checks for debug builds.", 
+            default = false, 
+            type = "boolean"
+	    })
+	    add_configs("signcode", {
+            description = "Digitally sign files.", 
+            default = false, 
+            type = "boolean"
+	    })
+    end
+
+    -- Everything except Windows options
+    if not is_plat("windows") then
+	    add_configs("mysqlcompat", {
+            description = "Creates libmysql* symbolic links.", 
+            default = false, 
+            type = "boolean"
+	    })
+    end
+
+    -- Options for all platforms
+    add_configs("dyncol", {
+	    description = "Enables support of dynamic columns.", 
+	    default = true, 
+	    type = "boolean"
+    })
+    add_configs("curl", {
+	    description = "Enables use of curl.", 
+	    default = true, 
+	    type = "boolean"
+    })
+    add_configs("external_zlib", {
+	    description = "Enables use of external zlib.", 
+	    default = false, 
+	    type = "boolean"
+    })
+    add_configs("ssl", {
+	    description = "Enables use of TLS/SSL library.", 
+	    default = true, 
+	    type = "boolean"
+    })
+    add_configs("unit_tests", {
+	    description = "Build test suite.", 
+	    default = false, 
+	    type = "boolean"
+    })
+
+    on_load(function (package)
+	    local configdeps = {curl           = "libcurl",
+	                        external_zlib  = "zlib",
+	                        ssl            = "openssl"}
+
+	    for name, dep in pairs(configdeps) do
+		    if package:config(name) then
+			    package:add("deps", dep)
+		    end
+	    end
+    end)
+
+    on_install("bsd", "linux", "windows", function(package)
+	    local configs = {}
```

Comment:
tab intend? please use 4 spaces instead of tab.

Reply:
I am using 4 spaces?

Reply:
This is my bad. I need to debug my editor config.